frontend/pages/users/index.tsx

```diff
@@ -0,0 +1,148 @@
+import UserTab from '@/components/molecules/UserTab'
+import Paginate from '@/components/organisms/Paginate'
+import { useState } from 'react'
+const initialUserList = [
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 10000,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 10000,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 10000,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 0,
+        role: 'Team Lead',
+    },
+    {
+        id: 1,
+        first_name: 'Ian',
+        last_name: 'Urriza',
+        avatar: '/../../public/images/sun_logo.png',
+        reputation: 10000,
+        role: 'Team Lead',
+    },
+]
+
+const roleFilters = [
+    { id: 1, name: 'Admin', onClick: function () {} },
+    { id: 2, name: 'Team Leader', onClick: function () {} },
+    { id: 3, name: 'Member', onClick: function () {} },
+]
+
+const scoreSort = [
+    { id: 1, name: 'Most Score', onClick: function () {} },
+    { id: 2, name: 'Least Score', onClick: function () {} },
+]
+
+const UsersPage = () => {
+    const userList = useState([])
```

Comment:
Please remove if not in being used

Reply:
I've removed it along with some of the imports